ugt_fwtools/synth_1_module.py

```diff
@@ -7,78 +7,72 @@
 import shutil
 import sys
 from . import utils
-from .synthesis import create_module, implement_module, show_screen_sessions, start_screen_session
+from .synthesis import create_module, implement_module, show_screen_sessions
 
-tty_bold_red = "\033[1;31m"
-tty_reset = "\033[0m"
-
-def print_error(message):
-    if sys.stdout.isatty():
-        message = f"{tty_bold_red}{message}{tty_reset}"
-    print(message)
 
 def parse_args():
     """Parse command line arguments."""
     parser = argparse.ArgumentParser()
-    parser.add_argument("mod_id", help="module number (eg. 1)")
-    parser.add_argument("path", metavar="<path>", type=os.path.abspath, help=f"fw build path to config file")
+    parser.add_argument("module_id", type=int, help="module ID (eg. 1)")
+    parser.add_argument("filename", type=os.path.abspath, help=f"build config file (*.cfg)")
     return parser.parse_args()
 
+
 def main() -> None:
     """Main routine."""
 
     # Parse command line arguments.
     args = parse_args()
-    
-    if not os.path.isfile(args.path):
-        message = f"\n===> no such file {args.path!r}\n"
-        print_error(message) 
-        raise RuntimeError() 
```

Comment:
Try to avoid raising exceptions without any message (no context).

Reply:
Done!